README-zh.md

```diff
@@ -23,6 +23,8 @@ $ git clone https://github.com/olOwOlo/hugo-theme-even themes/even
 
 **重要:** 在主题的 [`exampleSite`](https://github.com/olOwOlo/hugo-theme-even/tree/master/exampleSite) 目录下有一个 [`config.toml`](https://github.com/olOwOlo/hugo-theme-even/blob/master/exampleSite/config.toml) 文件，**将这个 [`config.toml`](https://github.com/olOwOlo/hugo-theme-even/blob/master/exampleSite/config.toml) 文件复制到你的站点目录下**，根据自己的需求更改即可。
 
+**重要:** 本主题用到了 [Hugo Pipes](https://gohugo.io/hugo-pipes/introduction/) 功能。如需修改 `asserts` 目录下的文件，请安装 extended 版。
```

Comment:
@olOwOlo Please correct the typo here. `asserts` should be `assets`. The English README should also be corrected.

Reply:
Fixed.